Fix Compilation issue in ,
	- vpe.c for incompatible types when assigning to type 'unsigned int' from type 'kuid_t' and 'kgid_t'
	- unsed variable rt in rtlx.c

diff --git a/arch/mips/kernel/rtlx.c b/arch/mips/kernel/rtlx.c
--- a/arch/mips/kernel/rtlx.c
+++ b/arch/mips/kernel/rtlx.c
@@ -438,7 +438,6 @@ static ssize_t file_write(struct file *f
 			  size_t count, loff_t * ppos)
 {
 	int minor = iminor(file_inode(file));
-	struct rtlx_channel *rt = &rtlx->channel[minor];
 
 	/* any space left... */
 	if (!rtlx_write_poll(minor)) {
diff --git a/arch/mips/kernel/vpe.c b/arch/mips/kernel/vpe.c
--- a/arch/mips/kernel/vpe.c
+++ b/arch/mips/kernel/vpe.c
@@ -1270,8 +1270,13 @@ static int vpe_open(struct inode *inode,
 	v->load_addr = NULL;
 	v->len = 0;
 
+#ifdef CONFIG_UIDGID_STRICT_TYPE_CHECKS
+	v->uid = filp->f_cred->fsuid.val;
+	v->gid = filp->f_cred->fsgid.val;
+#else
 	v->uid = filp->f_cred->fsuid;
 	v->gid = filp->f_cred->fsgid;
+#endif
 
 	v->cwd[0] = 0;
 	ret = getcwd(v->cwd, VPE_PATH_MAX);
